client/src/components/NewParkForm.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react"
+import translateServerErrors from "../services/translateServerErrors.js"
+import FormError from "./layout/FormError.js"
+import {Redirect} from "react-router-dom"
+
+const NewParkForm = (props) => {
+
+  const [newPark, setNewPark] = useState({
+    name: "",
+    description: "",
+    location: "",
+    rating: "",
+    picture: "",
+  })
+
+  const [errors, setErrors] = useState([])
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const postPark = async (newParkData) => {
+    try {
+      const response = await fetch(`/api/v1/parks`, {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(newParkData),
+      });
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json();
+          const newErrors = translateServerErrors(body.errors);
+          return setErrors(newErrors);
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`;
+          const error = new Error(errorMessage);
+          throw error;
+        }
+      } else {
+        const body = await response.json();
+        console.log("posted successfully", body)
+        setShouldRedirect(true)
+       
+      }
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+
```

Comment:
Please clean up whitespace here: have only one empty line here, and one in between `handleInputChange` and `handleSubmit`